### ISSUE TITLE
Are there any plans to define levels 3 and 4?

### ISSUE THREAD

**Issue by icedwater** [hongkonggong/tldr-digital-security#3]
I think going from levels 1 and 2 directly to level 5 is a bit odd; what are the definitions for the levels in between?

**Comment by hongkonggong** [comment]
the original intent was to highlight the fact that there was a big jump between accessible practices versus technical/esoteric practices that i discovered in the course of my research. (e.g. PGP email is kind of an all or nothing security practice.) i also mapped levels 1-2 as Things Everyone Should Do, and level 5 as Things People With Identified Threats Should Do.

does that make sense?

**Comment by icedwater** [comment]
Yeah, that kind of makes sense; still it's a little jarring to see the gap. It's true, there doesn't seem to really be a middle ground, so probably better to leave the space open until someone knows better. Hope to hear from the others who've read this :)

**Comment by hongkonggong** [comment]
Hi! Just wanted to follow up on this to see what you thought. So I've added a level 3, which has become more time consuming but still probably essential tasks. Then beyond that, I've broken them up into scenarios, because the recommendations are less monolithic/shared for these, and so there's no direct progression so to speak.

**Comment by icedwater** [comment]
Thanks! I'll look at it again :)
